tidb-cloud/tidb-cloud-console-auditing.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Web Console Audit Logging
+Summary: Learn about how to audit operations on the TiDB Cloud Web Console.
+---
+
+# Web Console Audit Logging
+
+TiDB Cloud provides you with the audit log function for Web Console operations, which can record the history of user access details (such as user login to Web Console, cluster creation, etc.).
+
+> **Note:**
+>
+> Currently, the **audit logging** feature is experimental. The interface and output are subject to change.
+
+To assess the effectiveness of user access policies and other information security measures of your organization, it is a security best practice to conduct a periodic analysis of the DbaaS audit logs.
+
+## Specify auditing filter rules
+
+The web console audit logging feature is enabled by default, and all user operations on the Web Console will be recorded in the audit logs.
+Audit logs cover the following event types:
+
+| Audit Event Type | Description |
+|---|---|
+| AuditEventSignIn | sign in |
+| AuditEventSignOut | sign out |
+| AuditEventUpdateUserProfile | update user's first name and last name |
+| AuditEventUpdateMFA | enable or disable MFA |
+| AuditEventCreateProject | create a new project
+| AuditEventUpdateProject | update the project name
+| AuditEventDeleteProject | delete a project
+| AuditEventInviteUserIntoProject | invite a user into project
+| AuditEventDeleteProjectUser | delete a project user
+| AuditEventUpdateOrg | update organization name and time zone
+| AuditEventCreateIntegration | create an integration
+| AuditEventDeleteIntegration | delete an integration
+| AuditEventListOrgUsers | list users in organization
+| AuditEventListProjectUsers | list users in a project
+| AuditEventAddNewPaymentMethod | add a new credit card
+| AuditEventUpdatePaymentMethod | update credit card info
```

Comment:
A `|` is missing at the right end of each line.

Reply:
updated